dashboard/next.config.js

```diff
@@ -20,16 +20,6 @@
  */
 const nextConfig = {
   trailingSlash: true,
-
-  rewrites: () => {
```

Comment:
No need for rewrites now as we directly send requests to the mock server with CORS allowed.